packages/gusb.rb

```diff
@@ -0,0 +1,25 @@
+require 'package'
+
+class Gusb < Package
+  description 'GUsb is a GObject wrapper for libusb1'
+  homepage 'https://www.openhub.net/p/gusb'
+  version '0.2.11'
+  source_url 'https://github.com/hughsie/libgusb/archive/gusb_0_2_11.tar.gz'
+  source_sha256 '090eb605e75f8a5b0b3df7ff29d96dd51730850ac89417378d4a8d39fab13702'
+
+  depends_on 'automake' => :build
```

Comment:
Do we also need `depends_on 'libtool'` or `depends_on 'intltool'` since we're executing autogen.sh?

Reply:
Could automake , libtool and intltool be merged into buildessential ?

Reply:
That is basically what I proposed in #1742.